### ISSUE TITLE
A new RAW Data export format that is Flat and has one row for each action: one row for each page view, each goal conversion...

### ISSUE THREAD

**Issue by Chardonneaur** [piwik/piwik#22531]
Hi team,

Currently if we take the visit log report, actions and orders are arrays.
As a result, it is impossible to create custom reports without coding.
The idea would be to have an option in order to make this report flat so that each action and each item become a single row for each.

**Comment by sgiehl** [comment]
But would it be helpful if you have tons of entries like `<action1>`, `<action2>`,.... or `<order1>`, `<order2>`,.... The available entries would then differ from record to record, which might not be more usable than now. 🤔

**Comment by Chardonneaur** [comment]
@sgiehl visually, it will looks like this https://youtu.be/q1PEiJT1qOE and solve big problems for data analysts.

**Comment by mattab** [comment]
@Chardonneaur i've just renamed the issue. Hopefully your intent is captured in this new title?

**Comment by Chardonneaur** [comment]
@mattab yes, as far as all the data are getting displayed in a flat format, i am fine with it. This simple change will significantly increase the user experience.

**Comment by Chardonneaur** [comment]
@mattab closing this ticket is a big thing, because typically for Google Looker Studio end users, it allows them to get exactly what they want.
